cli/azd/pkg/project/framework_service_npm.go

```diff
@@ -77,7 +77,7 @@ func (np *npmProject) Build(
 			// Exec custom `build` script if available
 			// If `build`` script is not defined in the package.json the NPM script will NOT fail
 			task.SetProgress(NewServiceProgress("Running NPM build script"))
-			if err := np.cli.RunScript(ctx, serviceConfig.Path(), "build", np.env.Environ()); err != nil {
+			if err := np.cli.RunScript(ctx, serviceConfig.Path(), "build", []string{}); err != nil {
```

Comment:
Did you purposely omit the `package` script?  When I first came in here I would have expected you to make a similar change there.  A more consistent change would be to just remove the `env` param on the `RunScript(...)` that they both consume.

Reply:
Can you check if the change below is the `package` script change you're referring to?

I think we still need the `env` for `NODE_ENV=production`, unless we're dropping that requirement.